cmd/crybapy/cmd_stat.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"context"
+	"encoding/csv"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/shopspring/decimal"
+	"github.com/spf13/cobra"
+	"github.com/zeebo/errs"
+
+	"storj.io/crypto-batch-payment/pkg/pipelinedb"
+)
+
+type statConfig struct {
+	rootConfig *rootConfig
+	Root       string
+}
+
+func newStatCommand(rootConfig *rootConfig) *cobra.Command {
+	config := &statConfig{
+		rootConfig: rootConfig,
+	}
+	cmd := &cobra.Command{
+		Use:   "stat",
+		Short: "Print out statistical information about payment",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) > 0 {
+				config.Root = args[0]
+			} else {
+				config.Root = "."
+			}
+			return checkCmd(doStat(config))
+		},
+	}
+
+	return cmd
+}
+
+func doStat(config *statConfig) error {
+	out := csv.NewWriter(os.Stdout)
+	defer out.Flush()
+	err := out.Write([]string{"file", "confirmed_txs", "all_txs", "payouts", "usd_amount", "used_gas"})
+	if err != nil {
+		return errs.Wrap(err)
+	}
+	ctx := context.Background()
+	return filepath.WalkDir(config.Root, func(path string, entry os.DirEntry, err error) error {
+		if entry.Name() == "payouts.db" {
+			err := printFileStat(ctx, out, path)
+			if err != nil {
+				fmt.Println("Ignoring", path, err.Error())
```

Comment:
Shouldn't this go to `stderr`? Otherwise it might get injected into the csv at a random point.

Reply:
Good point. Fixed.